packages/app/src/cli/services/dev/update-extension.ts

```diff
@@ -55,10 +55,10 @@ export async function updateExtensionDraft({
     serialized_script: encodedFile,
   }
   if (extension.isFunctionExtension) {
-    const compiledFiles = await readFile(extension.outputPath, {encoding: 'base64'})
+    const compiledFiles = await readFile(extension.outputPath)
```

Comment:
Do not need the base64 encoding because for this draftable flow, the bundlePath ends up being the same as `extension.outputPath` and we already have logic in place to write base64 encoded data to the bundlepath, so outputpath here already ends up having base64 encoded content.